src/main/java/pl/kremiec/registrationapp/controller/RestController.java

```diff
@@ -0,0 +1,59 @@
+package pl.kremiec.registrationapp.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.kremiec.registrationapp.model.User;
+import pl.kremiec.registrationapp.repo.UserRepo;
+import pl.kremiec.registrationapp.service.RestService;
+
+@Controller
+public class RestController {
+
+    private RestService restService;
+    private UserRepo userRepo;
+
+    public RestController(RestService restService, UserRepo userRepo) {
+        this.restService = restService;
+        this.userRepo = userRepo;
+    }
+
+    @GetMapping("")
+    public String welcome(){
+        return "welcomePage";
+    }
+
+    @GetMapping("/user/create")
+    public String register(Model model){
+        model.addAttribute("user", new User());
+        return "userCreate";
+    }
+
+    @PostMapping("/user/create")
+    public String submitForm(@ModelAttribute("user") User user, Model model) {
+        model.addAttribute("user", user);
+        if(restService.emailCheck(user.getEmail())){
```

Comment:
space before and after, please.
```suggestion
        if (restService.emailCheck(user.getEmail())) {
```